src/ucp/core/ucp_context.c

```diff
@@ -84,12 +89,19 @@ static ucs_config_field_t ucp_config_table[] = {
    "Estimation of buffer copy bandwidth",
    ucs_offsetof(ucp_config_t, ctx.bcopy_bw), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"ATOMIC_MODE", "device",
+   "Atomic operations synchronization mode.\n"
+   " host   - operations are atomic with respect to the host processor.\n"
```

Comment:
@shamisp the default value would be printed in ucx_info explicitly, no need to add it again:
```
#
# Atomic operations synchronization mode.
#  host   - operations are atomic with respect to the host processor.
#  device - atomic operations are performed on one of the transport devices,
#           and there is no atomicity guarantee with respect to the host processor.
#
# Syntax: [host|device]
#
UCX_ATOMIC_MODE=device
```